### ISSUE TITLE
Alerts and destinations should be exportable/importable objects

### ISSUE THREAD

**Issue by aimtsou** [opensearch-project/alerting#120]
**Is your feature request related to a problem? Please describe.**
We see that alerts and destinations are not considered objects and they cannot be exported/imported.

**Describe the solution you'd like**
Just export the alerts and destinations as json objects so we can import them either in another tenant or for future uasage.

**Describe alternatives you've considered**
No alternatives considered.

**Comment by skkosuri-amzn** [comment]
Thanks for creating this issue. I assume you meant **Monitors** and Destinations? 
We could support `GET _opendistro/_alerting/monitors` that gets all the monitors (export). Similarly `POST _opendistro/_alerting/monitors` to create multiple monitors (import).

**Comment by aimtsou** [comment]
Good afternoon,

yes you are right Monitors and Destinations(I am not sure if the dashboard could be done also).
They could belong to the saved objects in stack management like saved searches, graphs and dashboards. So you could export them and import them if needed.

I hope this clarifies the case. Also the console API could be used as an alternative.

**Comment by skkosuri-amzn** [comment]
Monitors, Desitnations are OpenSearch objects. They can be created/modified/deleted without OpenSearch Dashboards (Kibana). They are not like Saved Searches, Dashboards which are stored as OpenSearch Dashboards (Kibana) objects.

**Comment by aimtsou** [comment]
I understand the concept described. Considering them exportable / importable objects would facilitate the work in multi-tenancy.

**Comment by skkosuri-amzn** [comment]
Yes, we are looking into export/import of monitors to start with. We will post an update in few weeks.

**Comment by skkosuri-amzn** [comment]
First PR for this feature https://github.com/opensearch-project/alerting/pull/173

**Comment by skkosuri-amzn** [comment]
Adding two new API to export and import monitors.
`/_plugins/_alerting/monitors/export` 
`/_plugins/_alerting/monitorsimport`

**Comment by faabsen** [comment]
> Adding two new API to export and import monitors. `/_plugins/_alerting/monitors/export` `/_plugins/_alerting/monitorsimport`

Is this functionality already working? If so, where can I find the documentation?

**Comment by excalq** [comment]
In the UI (version 2.7), monitors can be exported, however it does not look like there's a way to import monitor JSON in the UI.
